qt/widgets/common/src/FunctionBrowser/FunctionBrowserUtils.cpp

```diff
@@ -105,7 +105,12 @@ splitConstraintString(const QString &constraint) {
         lowerBoundStr = QString::fromStdString(expr[1].str());
       }
       paramName = QString::fromStdString(expr[0].str());
-    } else {           // parameter is second
+    } else { // parameter is second
+      try {
```

Comment:
`try_lexical_convert` would be a better option here.

Exception shouldn't be used for non-exceptional control flow.

Reply:
Ah yes that would be better - I didn't know that existed!